ui/store/actions.ts

```diff
@@ -5085,6 +5086,42 @@ export function setName(
   }) as any;
 }
 
+/**
```

Comment:
Can you add a unit test for each of these?

Reply:
added